meta-iotqa/lib/oeqa/runtime/multimedia/opencv/opencv_cv2.py

```diff
@@ -56,28 +64,29 @@ def setUp(self):
                 OpenCVTest.cv_image_src),
             OpenCVTest.cv_image_src_target)
 
+    # Test that OpenCV can read image file from file system
```

Comment:
This could also be done as docstring, i.e.
```
def test_cv_read_image(self):
    """Test that OpenCV can read image file from file system"""
```

But that doesn't matter in practice, so don't bother changing it.